io.github.bkueng.qMasterPassword.yml

```
@@ -0,0 +1,36 @@
+id: io.github.bkueng.qMasterPassword
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: qMasterPassword
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
+  - name: libscrypt
+    buildsystem: autotools
+    no-autogen: true
+    make-install-args:
+      - PREFIX=/app
+    sources:
+      - type: archive
+        url: https://github.com/technion/libscrypt/archive/v1.22.tar.gz
+        sha256: a2d30ea16e6d288772791de68be56153965fe4fd4bcd787777618b8048708936
+  - name: qMasterPassword
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    post-install:
+      - mkdir -p /app/share/icons/hicolor/512x512/apps
+      - cp /app/share/pixmaps/qmasterpassword.png /app/share/icons/hicolor/512x512/apps/$FLATPAK_ID.png
+      - cp /app/share/metainfo/{qMasterPassword,$FLATPAK_ID}.appdata.xml
+      - sed -i "s/<id>qMasterPassword.desktop<\/id>/<id>$FLATPAK_ID<\/id>/g" /app/share/metainfo/$FLATPAK_ID.appdata.xml
+      - cp /app/share/applications/{qMasterPassword,$FLATPAK_ID}.desktop
+      - sed -i "s/Icon=qmasterpassword/Icon=$FLATPAK_ID/g" /app/share/applications/$FLATPAK_ID.desktop
```

```suggestion
      - cp /app/share/metainfo/{qMasterPassword,$FLATPAK_ID}.appdata.xml
      - sed -i "s/<id>qMasterPassword.desktop<\/id>/<id>$FLATPAK_ID<\/id>/g" /app/share/metainfo/$FLATPAK_ID.appdata.xml
      - cp /app/share/applications/{qMasterPassword,$FLATPAK_ID}.desktop
      - sed -i "s/Icon=qmasterpassword/Icon=$FLATPAK_ID/g" /app/share/applications/$FLATPAK_ID.desktop
```

Use rename-desktop-file, rename-appdata-file and rename-icon, see man flatpak-manifest. They will fix the id in appdata, icon in desktop etc. automatically.